.ci.yaml

```diff
@@ -2345,6 +2345,18 @@ targets:
       task_name: android_lifecycles_test
     scheduler: luci
 
+  - name: Linux_android openpay_benchmarks__scroll_perf
+    bringup: true
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    timeout: 60
+    properties:
+      openpay: "true"
+      tags: >
+        ["devicelab", "android", "linux", "openpay"]
```

Comment:
With the above comment, let's also add two more tags `samsung` and `a02`.

Reply:
Done!